src/transformers/utils/quantization_config.py

```diff
@@ -379,7 +379,9 @@ class GPTQConfig(QuantizationConfigMixin):
             The exllama config. You can specify the version of the exllama kernel through the `version` key. Defaults
             to `{"version": 1}` if unset.
         cache_block_outputs (`bool`, *optional*, defaults to `True`):
-                Whether to cache block outputs to reuse as inputs for the succeeding block.
+            Whether to cache block outputs to reuse as inputs for the succeeding block.
+        inside_layer_modules (`List`, *optional*):
+            List of module names to quantize inside block_name_to_quantize. If not set, we will quantize all the linear layers.
```

Comment:
the name is not intuitive 😅

Reply:
+1 on this. Also, I'm slightly confused about the intended data structures here and the desired logic. If `block_name_to_quantize` is not set and `inside_layer_modules` is not None, does this still control the layers that are quantized? What if there's more that one block you want to quantize? Would this be a list of lists?

Reply:
Sorry for that. I reused the naming on autoGPTQ. I changed it to `modules_to_quantize_inside_block`. Concerning `block_name_to_quantize`, if we are unable to infer the block_name, we will return an error. So `block_name_to_quantize` will be set in any case.